components/lib/nearby/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,7 @@
+<!--
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~  License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+  -->
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="mozilla.components.lib.push.amazon" />
```

Comment:
nit: package name -> mozilla.components.lib.nearby

Reply:
Whoops!

Reply:
@espertus this might have been the reason ktlint wasn't working earlier for you. :)